community/kernel/src/test/java/org/neo4j/kernel/impl/store/format/RecordFormatTest.java

```diff
@@ -64,7 +64,7 @@
     public static final RandomRule random = new RandomRule();
 
     private final EphemeralFileSystemRule fsRule = new EphemeralFileSystemRule();
-    private final PageCacheRule pageCacheRule = new PageCacheRule( false /*true here later*/ );
+    private final PageCacheRule pageCacheRule = new PageCacheRule( true );
```

Comment:
Maybe add a comment or name the literal value in a variable to explain what the `true` means here?